source/ecloud/VPC/VPC's.md

```diff
@@ -0,0 +1,15 @@
+# Virtual Private Cloud
+
+A VPC is your virtual Private Cloud from which you can create Virtual Networks, Instances and more. To create a VPC select the VPC option from the menu. Click the 'Create VPC' button which will take you to the setup page.
+
+You can give your VPC an identifying name here and the region in which you want to create the VPC. The regions available are Manchester, Amsterdam or London.
+
+After you have given this a name and selected a region you can choose either an Automated Network deployment which will automatically create a network for you or deselect this option to manually [create a Network.](Network doc to be placed here) 
```

Comment:
Do you have the link to place here now @walshy90? If not, can we remove this link and add it back in when you do have the linked page?